source/apps/change-deployment-models.txt

```diff
@@ -30,6 +30,15 @@ a version that supports changing deployment models. If you change
 deployment models before upgrading, the SDK will not be able to connect
 and requests will fail.
 
+The following SDK versions support changing deployment models:
```

Comment:
We have a similar SDK version support list on the [Migrate Device Sync mode](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-31122/sync/migrate-sync-modes/#requirements) page. The source for the page is here: https://github.com/mongodb/docs-app-services/blob/65e6e429c5c906d80c5911b4cad675dfaa0a1c74/source/sync/migrate-sync-modes.txt#L43

It would probably be good to present this list formatted similar to that one for consistency. So, for example, C++ would be:

`- Realm C++ SDK v0.2.0`

And so on.

Also, for JS, we would use the full marketing name for both SDKs that use it - i.e.:

```
- Realm Node.js SDK v12.0.0
- Realm React Native SDK v12.0.0
```

Reply:
ah, i was wondering if we had anywhere where we listed out SDK versions that i might be able to match in form – thank you, Dachary!